report_qweb_txt/models/report.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+
+class Report(models.Model):
+    _inherit = "report"
+
+    @api.model
+    def _get_report_from_name(self, report_name):
```

Comment:
call `super` here and do your search only if it's result is falsy. And given `super` doesn't raise if nothing is found, this function shouldn't either

Reply:
done.